optaplanner-examples/src/main/java/org/optaplanner/examples/vehiclerouting/persistence/VehicleRoutingImporter.java

```diff
@@ -391,6 +391,8 @@ private void createVehicleList() {
                 vehicle.setId(id);
                 id++;
                 vehicle.setCapacity(capacity);
+                //Multiple Depot VRP assign one vehicle for each depot
+                depotList.get(i % depotList.size());
```

Comment:
@madhanpavan The point is that the value returned from depotList.get() isn't used and since ArrayList.get() and ArrayList.size() methods have no side effects the line indeed does nothing (unless I'm missing something). Did you instead want to write:
```java
vehicle.setDepot(depotList.get(i % depotList.size()));
```
or something similar?